tests/integration/reqs/test_feature.py

```diff
@@ -0,0 +1,29 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+from tests.integration.it_utils import test_async_and_sync
+from xrpl.models.requests import Feature
+
+AMENDMENT = "8CC0774A3BF66D1D22E76BBDA8E8A232E6B6313834301B3B23E8601196AE6455"
+
+
+class TestAMMInfo(IntegrationTestCase):
```

Comment:
There already exists a `class TestAMMInfo` [here](https://github.com/XRPLF/xrpl-py/blob/c4913b56d807753bf57eef36854d92c8f4b9bd2f/tests/unit/models/requests/test_amm_info.py#L10)

Instead, this could be named as `TestFeatureRPC` or something like that

Reply:
typo